sys/net/network_layer/fib/fib.c

```diff
@@ -956,7 +967,8 @@ int fib_sr_entry_append(fib_table_t *table, fib_sr_t *fib_sr,
     fib_sr_entry_t *elt = NULL;
     LL_FOREACH(fib_sr->sr_path, elt) {
         size_t addr_size_match = addr_size << 3;
-        if (universal_address_compare(elt->address, addr, &addr_size_match) == UNIVERSAL_ADDRESS_EQUAL) {
+        if (UNIVERSAL_ADDRESS_EQUAL == universal_address_compare(elt->address, addr,
+                                                                 &addr_size_match)) {
```

Comment:
I'm not complaining about the change, but CI is warning about *that*?
Yoda conditions can be helpful, but do we really need CI to enforce them? Assignments inside `if` conditions already raise a warning, so this seems unnecessary.